pinot-tools/src/main/java/org/apache/pinot/tools/admin/command/QuickstartRunner.java

```diff
@@ -204,11 +209,30 @@ public void createBrokerTenantWith(int number, String tenantName)
 
   public void bootstrapTable()
       throws Exception {
-    for (QuickstartTableRequest request : _tableRequests) {
-      if (!new BootstrapTableTool("http", InetAddress.getLocalHost().getHostName(), _controllerPorts.get(0),
-          request.getBootstrapTableDir(), _authToken).execute()) {
-        throw new RuntimeException("Failed to bootstrap table with request - " + request);
+    ExecutorService executorService = Executors.newCachedThreadPool();
```

Comment:
I feel the retry might not be needed, and won't really help if the controller stuck. Adding another executor will add more overhead to the CPU

Reply:
I'm trying to find where the stuck comes from.
If a client-side issue, then retry will help.
Otherwise, it's a controller HTTP server-side issue, we need to fix it.
GithubAction gives 2 cpu and 7GB ram, so I feel it should be enough for quickstart.